docs/user-guide.md

```diff
@@ -101,10 +101,32 @@ kedge version
 
 ## `kedge init`
 
-Initialize kedge file
+### Basic getting started
 
-```sh
-kedge init --file kedge.yml --name web --image centos/httpd --port 80
+```bash
+kedge init --name web --image centos/httpd --ports 80
+```
+
+This will create a `kedge.yml` file for a long running application running inside
```

Comment:
Why do we need "long running" here?
How about, "file for an httpd web server with container image centos/httpd"

Reply:
done